### PR TITLE
New ld_library_path_of_user spec and UserLdLibraryPath parser

### PR DESCRIPTION
**Pull request by xiangce**

### CODE REVIEW

File: insights/specs/default.py
Comment (bfahr) [inline insights/specs/default.py:525]: @xiangce should you use the `env` command instead of `export` to ensure that it will work regardless of the shell configuration?

File: insights/parsers/ld_library_path.py
Comment (bfahr) [inline insights/parsers/ld_library_path.py:71]: @xiangce this parser didn't work and it was here for such a short period of time I would just remove this parser instead of deprecating it.

**Review by bfahr — CHANGES_REQUESTED** [review]

File: insights/specs/default.py
Comment (xiangce) [reply insights/specs/default.py]: Done

**Review by bfahr — APPROVED** [review]
Approved pending approval of the spec.

**Pull request merged**
